DataProc/load.py

```diff
@@ -0,0 +1,25 @@
+import mypy
+
+import skvideo.io as sk 
+import wandb
+
+import numpy as np
+
+
+class VideoLoader(object):
```

Comment:
Is this class intended to be ever initialized? If not - please make it abstract. Attach docstring specifying its purpose.